usaspending_api/disaster/v2/views/disaster_base.py

```diff
@@ -1,12 +1,57 @@
 from django.utils.functional import cached_property
 from rest_framework.views import APIView
 
+from django.db.models import Exists, OuterRef, Max
 from usaspending_api.awards.v2.lookups.lookups import award_type_mapping
 from usaspending_api.common.data_classes import Pagination
 from usaspending_api.common.validator import customize_pagination_with_sort_columns, TinyShield
 from usaspending_api.references.models import DisasterEmergencyFundCode
+from usaspending_api.references.models.gtas_sf133_balances import GTASSF133Balances
+from usaspending_api.awards.models.financial_accounts_by_awards import FinancialAccountsByAwards
 from usaspending_api.submissions.helpers import get_last_closed_submission_date
 
+COVID_19_GROUP_NAME = "covid_19"
+
+
+def covid_def_codes():
+    return DisasterEmergencyFundCode.objects.filter(group_name=COVID_19_GROUP_NAME)
+
+
+def covid_def_code_strings():
+    return [code["code"] for code in covid_def_codes().values("code")]
+
+
+def latest_gtas_of_each_year():
```

Comment:
Might be a good idea to add some indication in the function name that this returns a Django queryset. Same with `latest_faba_of_each_year()`